### PR TITLE
solution

### PR DESCRIPTION
**Pull request by taras-zelinskyi**

### CODE REVIEW

File: services/user.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        password=password
+    )
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+    return user
```
Comment (Abnormaltype) [inline services/user.py:24]: unnecessary `return`

File: services/user.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    if password:
+        user.set_password(f"{password}")
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+    return user
```
Comment (Abnormaltype) [inline services/user.py:51]: the same

File: services/user.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+def update_user(
+        user_id: int,
+        username: str = None,
+        password: str = None,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    user = get_user_model().objects.get(id=user_id)
```
Comment (Abnormaltype) [inline services/user.py:39]: why so you need `get_user`?

File: services/user.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    if username:
+        user.username = username
+    if password:
+        user.set_password(f"{password}")
```
Comment (Abnormaltype) [inline services/user.py:43]:
```suggestion
        user.set_password(password)
```

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

File: services/user.py (hunk 4 above)
Comment (taras-zelinskyi) [reply services/user.py:43]: Created new pull request

**Pull request closed without merging**